internal/configs/node/config.go

```diff
@@ -178,6 +179,16 @@ func (conf *ConfigType) SetPushgatewayPort(port string) {
 	conf.PushgatewayPort = port
 }
 
+// SetMetricsFlag set the metrics flag
```

Comment:
@LeoHChen I'm a bit confused whether nodeConfig is being deprecated or not.
Maybe I misheard, but thought shardingconfig is where new configurations should be added (which also made it confusing if configs unrelated to sharding specifically should also be put there)

Could you clarify?

Reply:
nodeConfig is mostly used when we have command-line flags.  shardingconfig is not for command-line, those are built-in parameters that shouldn't be changed in command line.